bin/crew

```diff
@@ -1625,10 +1623,18 @@ def update_sha256(package, key, value)
 end
 
 def copy_package(pkg_name, prompt_msg = '')
+  pkg_file = File.join(CREW_LOCAL_REPO_ROOT, 'packages', "#{pkg_name}.rb")
+  # Use rubocop to sanitize package file, and let errors get flagged.
+  if @device[:installed_packages].any? { |elem| elem[:name] == 'ruby_rubocop' }
+    puts "Using rubocop to sanitize #{pkg_file} .".orange
+    system "rubocop -c #{File.join(CREW_LOCAL_REPO_ROOT, '.rubocop.yml')} -A #{pkg_file}", exception: true
```

Comment:
After thinking about this for a bit, I'm not so sure the `-A` option to actually change the file is a good idea.  Maybe a check and report would be better?  The reason I say this is because rubocop might get it wrong and it's not obvious to the user that the package was updated.

Reply:
I thought the point of this was to make things easier for new users-- I assume automatically fixing errors would be better, but I'm not too opinionated either way.

Reply:
It is definitely an opinionated approach but we to need to think about the best way to do this without making it confusing.  I'll go ahead and merge this to see how it works out and if it becomes an issue, we can change the process then.